Packages/com.unity.multiplayer.samples.coop/Utilities/SceneManagement/SceneLoader.cs

```diff
@@ -0,0 +1,141 @@
+using System.Collections.Generic;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Unity.Multiplayer.Samples.Utilities
+{
+    public class SceneLoader : NetworkBehaviour
```

Comment:
Non blocker nitpicking: That name tickles me. When I first saw it used in game code, I was wondering "why isn't Netcode's scene manager used anymore?"
and then I finally understood SceneLoader is a convenience wrapper that adds loading screen management to normal scene loading calls.
Maybe SceneLoaderWrapper? LoadingScreenManager?
I don't know, naming things is hard :/

Reply:
It is hard :( I probably spent more time trying to name this class than coding this entire PR. It has this name because my original intention was to create a generic wrapper for loading scenes because there is no callback I could attach to to start the loading screen when offline. But in the end, the only use it has is to tell the loading screen when to start and when to stop.

I think SceneLoaderWrapper would make more sense. LoadingScreenManager would create confusion with those wrapper methods I think, since from the outside they still load scenes first.

Reply:
I'm open to other names too, SceneLoaderWrapper isn't great either